paasta_tools/setup_chronos_job.py

```diff
@@ -161,6 +161,44 @@ def setup_job(service, instance, complete_job_config, client, cluster):
     )
 
 
+def config_with_historical_stats(chronos_client, service, instance, job_config):
+    """
+    Given a complete_job_config, return a new chronos_job_config
```

Comment:
if i understand correctly this isn't returning new config, but modifying the one passed to the function. i would very much prefer it to return a copy but changing the docstring is fine too.